### ISSUE TITLE
APK not exported anymore after upgrade to cordova-android 7.0.0

### ISSUE THREAD

**Issue by adrianmarinica** [bitrise-community/steps-cordova-archive#22]
I was using version 1.0.1 of `steps-cordova-archive` with `cordova-android v6.3.0` and everything was working properly. I needed to upgrade a plugin which required the move to `cordova-android v7.0.0`.

After this, the step stopped exporting the APK for the app build. I then proceeded to updated the step to version 1.1.0. This made it so that I got a zip archive exported (`APPNAME.app.dSYM.zip`). When downloaded, this archive is empty.

In the build logs, I see that the APK is actually created, probably there is an issue exporting it.

I have attached the final part of the logs, maybe it helps. I have sanitized them by changing the name of the app to `APPNAME` and the internal path to `APPPATH`.

```
Built the following apk(s): 
	/Users/vagrant/git/APPPATH/platforms/android/app/build/outputs/apk/debug/app-debug.apk
No scripts found for hook "after_compile".
Collecting ios outputs
The ipa path is now available in the Environment Variable: BITRISE_IPA_PATH (value: /Users/vagrant/deploy/APPNAME.ipa)
The dsym dir path is now available in the Environment Variable: BITRISE_DSYM_DIR_PATH (value: /Users/vagrant/deploy/APPNAME.app.dSYM)
The dsym.zip path is now available in the Environment Variable: BITRISE_DSYM_PATH (value: /Users/vagrant/deploy/APPNAME.app.dSYM.zip)
Output: /Users/vagrant/git/APPPATH/platforms/ios/build/device/APPNAME.app is a symlink to: /Users/vagrant/Library/Developer/Xcode/DerivedData/APPNAME-aemaofdoazmwwnatrxkqujeovfoa/Build/Intermediates.noindex/ArchiveIntermediates/APPNAME/InstallationBuildProductsLocation/Applications/APPNAME.app
Failed to export apps, error: resolved path: /Users/vagrant/Library/Developer/Xcode/DerivedData/APPNAME-aemaofdoazmwwnatrxkqujeovfoa/Build/Intermediates.noindex/ArchiveIntermediates/APPNAME/InstallationBuildProductsLocation/Applications/APPNAME.app does not exist

```

If you want me to share any other part of the logs, let me know. Thanks!

**Comment by viktorbenei** [comment]
Thanks for reporting - we'll check this ASAP! :)

**Comment by godrei** [comment]
Hi @adrianmarinica ,
thanks for the report!
We released a new version ([1.1.1](https://github.com/bitrise-community/steps-cordova-archive/releases/tag/1.1.1)) which fixes this issue.
I'll close this issue, but if you need more info/help just let us know and we'll reopen it!

**Comment by adrianmarinica** [comment]
I can confirm that everything works now. Thanks, guys!
